apps/web/src/screens/Dashboard/index.tsx

```diff
@@ -28,6 +28,8 @@ export const DashboardScreen: FC = () => {
 		await router.navigate('/');
 	};
 
+	localStorage.setItem('pathname', pathname);
+
```

Comment:
I think this feature could be reusable on mobile, thus we should not use `localStorage` here. You could try to store the layout `id` in setting document in DB and then navigate based on that `id`.

Reply:
do we need to synchronize the `layout_id` between mobile and extension?

Reply:
> I think this feature could be reusable on mobile, thus we should not use `localStorage` here. You could try to store the layout `id` in setting document in DB and then navigate based on that `id`.

btw, do we need to store screen like "setting" or "profile"

Reply:
> do we need to synchronize the `layout_id` between mobile and extension?

do you mean synchronize last project `id` between mobile and extension? I think those platforms share the feature not the data

Reply:
> > I think this feature could be reusable on mobile, thus we should not use `localStorage` here. You could try to store the layout `id` in setting document in DB and then navigate based on that `id`.
> 
> btw, do we need to store screen like "setting" or "profile"

I can't think of any purpose to save `setting` or `profile` screens, but we could check again with others.